.travis.yml

```diff
@@ -145,7 +150,7 @@ jobs:
     # -------------------------------------------------------------------------
     - stage: Deployment
       name: Deploy to docs.kuzzle.io
-      if: branch = master
+      if: branch = master AND (tag IS present)
```

Comment:
```suggestion
      if: tag IS present AND type != cron
```

Reply:
`branch = master` is optional here.

Reply:
nope, you only want `master` do be deployed on `docs.kuzzle.io` not the others.

Reply:
ah you mean  tags can only be applied to `master`? I'm not sure

Reply:
I confirm you can tag anything on any branch